components/esp_eth/include/esp_eth.h

```diff
@@ -136,6 +136,7 @@ typedef struct {
         .mac = emac,                     \
         .phy = ephy,                     \
         .check_link_period_ms = 2000,    \
+        .auto_nego_en = true,            \
```

Comment:
Thank you for pointing this issue out. This is really a mistake. However, proper fix should be removing `auto_nego_en` member from `esp_eth_config_t` structure.  `auto_nego_en` configuration member is not used anywhere and auto-negotiation should be only configurable via `esp_eth_ioctl` function from user's perspective. Could you please fix it in that way?

Reply:
Sure, I will push that in the next commit. Thanks!

Reply:
Thank you for the fix. However, could you please squash the commits into one as stated in the link provide above?

Reply:
I squashed the commits now.